cinema_service/urls.py

```diff
@@ -6,4 +8,4 @@
     path("api/cinema/", include("cinema.urls", namespace="cinema")),
     path("api/user/", include("user.urls", namespace="user")),
```

Comment:
Ensure that the `user.urls` module has a `app_name` defined to support the use of the `namespace` argument in the `include` function.